larq/layers_test.py

```diff
@@ -77,16 +77,17 @@ def test_binarization(self, quantized_layer, layer, input_shape, kwargs):
         input_data = random_input(input_shape)
         random_weight = np.random.random() - 0.5
 
-        quant_output = testing_utils.layer_test(
-            quantized_layer,
-            kwargs=dict(
-                **kwargs,
-                kernel_quantizer="ste_sign",
-                input_quantizer="ste_sign",
-                kernel_initializer=tf.keras.initializers.constant(random_weight),
-            ),
-            input_data=input_data,
-        )
+        with lq.metrics.scope(["flip_ratio"]):
```

Comment:
I do think we should have unit test for the `metrics` kwarg as well

Reply:
I added a unit test in the latest commit